lib/routes/index.js

```diff
@@ -30,6 +31,24 @@ module.exports = function routes(app, options, done) {
       res.end('ok');
     });
 
+    app.routes.post('/catchup', function catchup(req, res) {
```

Comment:
who or what service will be calling this route?

Reply:
the idea was that we either have a cron like thing that curls this route to trigger this or we use the setInterval feature. Im not sure what would be better, the setInterval way just makes it self managed.

Reply:
It's also generally useful from an operations standpoint. The API needs a public way to say _"hey, see if all the builds are properly queued."_